openapi.yaml

```diff
@@ -537,8 +537,6 @@ paths:
                             destination: ''
                           - name: Fred Smith
                             info: Out bound SMS text message sent OK.
-                            link:
-                              - null
```

Comment:
This `null` value was causing the Run Spectral action to fail - [null values are not supported by JSON-Parser](https://stackoverflow.com/a/34425408)